examples/live/check_results.py

```diff
@@ -6,98 +6,202 @@
+                        test_s1z = close(temp_s1z[trig_temp_id], trig_s1z, 1e-7)
+                        test_s2z = close(temp_s2z[trig_temp_id], trig_s2z, 1e-7)
+                        test_all = np.logical_and.reduce((test_mass1, test_mass2,
+                                              test_s1z, test_s2z))
+                        if not test_all.all():
+                              log.error('Invalid template parameters in %s %s',
+                                    trigfp, detector)
+                              single_fail = True
+
+      # check that triggers were produced in all detectors
+      if detectors_with_trigs != tested_detectors:
+            missing = sorted(tested_detectors - detectors_with_trigs)
+            log.error('No triggers found in %s', ', '.join(missing))
+            single_fail = True
+      
+      if single_fail: log.info('Single Trigger Test Failed')
+      return single_fail
+
+def check_coinc_results(inj_table):
+      coinc_fail = False
+      # gather coincident triggers
+      coinc_trig_paths = sorted(glob.glob('output/coinc*.xml.gz'))
+      l=len(coinc_trig_paths)
+      if l==0:
+            log.error('No coincident triggers detected')
+            coinc_fail = True
+      elif l>=10: 
+            log.error('Too many coincident triggers detected')
+            coinc_fail = True
+      else: 
+            log.info(str(l)+' coincident trigger(s) detected')
+      
+      #get properties of injection
+      inj_mass1=inj_table.get_column('mass1')[0]
+      inj_mass2=inj_table.get_column('mass2')[0]    
+      inj_spin1z=inj_table.get_column('spin1z')[0] 
+      inj_spin2z=inj_table.get_column('spin2z')[0] 
+      inj_time=inj_table.get_column('end_time')[0]
+      inj_snr=15
+      
+      #print properties of coincident triggers
+      for ctrigfp in coinc_trig_paths:            
+            xmldoc = ligolw_utils.load_filename(
+                  ctrigfp, False, contenthandler=LIGOLWContentHandler)
+            sngl_inspiral_table = lsctables.SnglInspiralTable.get_table(xmldoc)    
+            end_time=sngl_inspiral_table.get_column('end_time')  
+            snr_list=sngl_inspiral_table.get_column('snr')
+            network_snr_squared=0
+            for snr in snr_list:
+                  network_snr_squared+=snr**2
+            network_snr=np.sqrt(network_snr_squared)
+            mass1=sngl_inspiral_table.get_column('mass1')[0]
+            mass2=sngl_inspiral_table.get_column('mass2')[0]    
+            spin1z=sngl_inspiral_table.get_column('spin1z')[0] 
+            spin2z=sngl_inspiral_table.get_column('spin2z')[0] 
+            log.info('IFO SNRs: '+str(snr_list))
+            log.info('Network SNR: '+str(network_snr))
+            log.info('IFO End Times: '+str(end_time)) 
+            log.info('Mass 1: '+str(mass1))
+            log.info('Mass 2: '+str(mass2))
+            log.info('Spin1z: '+str(spin1z))
+            log.info('Spin2z: '+str(spin2z))
```

Comment:
I think it is not necessary to print all these info, but up to you. If you *do* keep these lines, please use the string format as suggested above (`%f` for float variables).

Reply:
This is prep for a possible future where the injection doesn't exactly match the template and we want to check which template is picking up the injection. I can however add an option to not print these out unless its wanted.